runtime/plugin/rplugin.vim

```diff
@@ -1,5 +1,16 @@
-if exists('g:loaded_remote_plugins') || &cp
+if exists('g:loaded_remote_plugins')
   finish
 endif
 let g:loaded_remote_plugins = 1
-call remote#host#LoadRemotePlugins()
+
+command! UpdateRemotePlugins call remote#host#UpdateRemotePlugins()
+
+augroup nvim-rplugin
+  autocmd!
+  autocmd FuncUndefined *
+        \ call remote#host#LoadRemotePluginsEvent(
+        \   'FuncUndefined', expand('<amatch>'))
+  autocmd CmdUndefined *
+        \ call remote#host#LoadRemotePluginsEvent(
+        \   'FuncUndefined', expand('<amatch>'))
```

Comment:
It listens to `CmdUndefined `, but sends `'FuncUndefined'`. What's this needed for?